### DISCUSSION TITLE
Not sure if I'm doing something wrong re: external resources

### DISCUSSION THREAD

**Discussion by RThomasHyde** [live-codes/livecodes#627] [General]
I wanted to try out something with the reactflow library, so I started up a project with the React Starter template. As soon as I added the reactflow (js & css) external resources, the console shows a "Script error" (I'd paste it here, but I don't seem able to copy from the console: as soon as I select the text it disappears and doesn't get copied if I ctrl-c). Then, when I add the jsx code from the [reactflow quick start](https://reactflow.dev/learn#creating-your-first-flow), I get a "TypeError: Cannot read properties of null (reading 'useRef')" error in the console and nothing renders. 

Is it me doing something wrong (I'm pretty new to this stuff, so I guess it's likely), or have I run into a livecodes issue (or known unsupported functionality that I failed to notice)?

**Comment by hatemhosny** [comment]
Thank you for reporting this.

This is a problem with resolving the modules from esm.sh from which modules are imported by default.
I have already reported [this issue there](https://github.com/esm-dev/esm.sh/issues/861).

I'm also aware about inability to copy from integrated console. I will also report that.

Thank you.

**Comment by RThomasHyde** [comment]
Thank you for the quick response! I look forward to a fix from the esm.sh maintainers, and will keep an eye out.

**Comment by hatemhosny** [comment]
The error occured because some react libraries imported react and LiveCodes playground imported another instance.
This should be fixed by #724 

Thank you @RThomasHyde for reporting that.

**Reply by RThomasHyde** [reply]
No problem, I'm excited that there's a fix!
